arnold/plugins/gaffer.mtd

```diff
@@ -267,6 +267,59 @@
 	shaderType STRING "lightFilter"
 	gaffer.nodeMenu.category STRING "Light"
 
+	gaffer.layout.section.Top.collapsed BOOL false
```

Comment:
Some prefer circular ordering, but personally find Top-Left-Right-Bottom more natural. Just food for thought.

Reply:
Yeah, I don't feel strongly about it either way. I went with the ordering that Arnold seems to deem reasonable. `kick -info barndoor` shows the parameters in this ordering.